### DISCUSSION TITLE
Is the Prometheus monitoring service still being maintained?

### DISCUSSION THREAD

**Discussion by scitarust** [microsoft/FluidFramework#9850] [Q&A]
Hello everyone
 
I noticed that the repository contains some code that tries to implement monitoring services. 
eg: 
  server/routerlicious/kubernetes/metric/
  server/charts/fluid-metrics/
But it seems that these codes can't run.

Has anyone successfully deployed a monitoring service? thanks

**Comment by tylerbutler** [comment]
@yangg-msft Is Azure Fluid Relay using these r11s services or anything based on them?
